packages/manager/modules/pci/src/components/hds/index.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+
+import component from './component';
+
+const moduleName = 'hdsOption';
+
+angular
+  .module(moduleName, ['pascalprecht.translate', 'oui'])
```

Comment:
```suggestion
  .module(moduleName, ['pascalprecht.translate', 'oui', 'ui.router'])
```

Reply:
I implement HDS as component without state strategy. 
So, I am not sure `ui.router` is required.

Reply:
My bad then 😄